### PR TITLE
Finishing NYArticle Clone

### PR DESCRIPTION
**Pull request by kiranitor123**
Completed the clone of the NYArticle

### CODE REVIEW

**Review by LamiaSristy — CHANGES_REQUESTED** [review]
# Status: Change Required. ♻️

Hello, @kiranitor123. 

**A lot of efforts and dedication needs for coding, And all are here present in your project.**

## The highlights:

* Your `git-flow` is correct. 💯
* Setting up `linters` is correct. 💯
* Nice work on embedded images and video. ⭐
* Proper use of the float, flexbox, and CSS grid. 👍 

## What to do next:
**In order to get approval please remember all the required checklist needs to add.**
**To compare please check the [link](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)**

## [REQUIRED]

- [x] Please add a short description to the Pull Request.
- [x] You are using `Bad Path` to import images and style-sheet. A good path starts with `./`. Please add this to all your images and style-sheet.


**README**

- [x]  Add a screenshot of the project in the browser.
- [x]  Add details of author.
- [x] In the Getting Started section add instructions for download and locally run your project.

**NavBar**
- [x] Add a `Hamburger button`  on the leftmost side.
- [x] Add a `search icon` 🔍  on the leftmost side. 
- [x] The `The NY Times logo` in the center is a link.
- [x] Add a border-bottom.

**Main Content**
- [x] For social media icons, please use images, font-awesome icons, or favicons.
- [x] The image-description for the first image, you need to make it align with the image, consider using `figure & figcaption` tag for this section.
- [x] For the big paragraph sections use `article` tag.
- [x]  In `**The Theory of Inflation**` don't use screenshots. Design this section using `CSS-grid`.

**Bottom part**
- [x] In the Footer section everything is a link. check the image below 👇 
ss6


**CSS**
- [x]  To follow the DRY rule and avoid using the same codes multiple times. You can create a common class to achieve this.

## 

**Make the changes and again ask for a re-review.**

Feel free to reach me on slack @lamiasristy 🙋
Keep up the spirit always. 🙌 
Happy Coding 💻

File: main.css [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+.title {
+  width: 600px;
+  margin: auto;
+}
+
+.social {
+  justify-content: center;
+  display: flex;
+}
+
```
Comment (LamiaSristy) [inline main.css]: instead of repeating display: flex; make a class named .display-flex { display: flex; } and use it everywhere.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  justify-content: space-between;
+  padding: 0;
+}
+
+.readMore {
+  display: flex;
+  justify-content: stretch;
+  margin: 20px 0;
+}
+
```
Comment (LamiaSristy) [inline main.css]: Use  .display-flex { display: flex; }.

File: main.css [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+.grid-container {
+  display: grid;
+  grid-template-columns: auto auto auto;
+}
+
+.grid-item {
+  padding: 10px;
+}
```
Comment (LamiaSristy) [inline main.css]: Make a class for padding: 10px; and use it. Do not repeat the same code,

File: index.html [hunk 4]
```diff
@@ -0,0 +1,660 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NYA Clone</title>
+    <link rel="stylesheet" href="main.css">
```
Comment (LamiaSristy) [inline index.html]: `<link rel="stylesheet" href="./main.css">`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,660 @@
...
+        <div>
+            <h1 class="title">Space Ripples Reveal Big Bang’s Smoking Gun</h1>
+        </div>
+        <div class="social">
+            <img src="assets/social1.png" alt="">
+        </div>
+        <div class="imagePrimary">
+            <img id='firstImage' src="assets/1.jpg" alt="main">
+
+        </div>
```
Comment (LamiaSristy) [inline index.html]: `something`

**Comment by kiranitor123** [comment]
Doing the NYC Article using HTML and CSS only, starting from scratch

File: main.css [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+  padding: 5px;
+}
+
+.contentVideo {
+  font-size: 15px;
+}
+
+.subscribeContent {
+  align-items: center;
+}
```
Comment (andresporras3423) [inline main.css]: Group selectors on lines 183 and 195

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Rolando Alvarez Faye** 
Amazing job with the improvements for this project :muscle:  :muscle:
Your page really looks like the original :clap: :clap: :clap:  
Just one last issue to correct before you can go ahead:

- [x] Group selectors with the same style definition. Check out this link: http://www.websiteoptimization.com/speed/tweak/groupsel/
 
Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

**Review by teekaytech — APPROVED** [review]
# Status: Approved :green_circle: 

**Congratulations @kiranitor123 :tada: this project has passed all the mandatory requirements. It is therefore approved :heavy_check_mark:**

**Comment(s)**
- In your next project, you should let your pull request summary be more descriptive.
- You should also learn to use a befitting title for your PR. You can simply say: `NYArticle Clone`.

Again, congratulations, you can merge to main/master.
Do not forget to fill the project completion form.
Good luck with your next task.

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Pull request merged**